contracts/test/CompoundGovernor.t.sol

```diff
@@ -418,6 +421,11 @@ contract HasVoted is CompoundGovernorTest {
 
         assertEq(governor.hasVoted(_proposalId, _voter), _hasVotedBefore ? true : false);
     }
+
+    function testFuzz_ZeroReturnedIf_HasVoted_Called_With_Invalid_ProposalId(uint256 _invalidProposalId, address _voter) public view {
+        vm.assume(_invalidProposalId != 0);
+        assertFalse(governor.hasVoted(_invalidProposalId, _voter));
```

Comment:
is this test name correct? it's returning `false`, right?

Reply:
Done: a700b5617b3a1f4253a9405e319740f24fd825e8